3p/integration.js

```diff
@@ -138,9 +138,14 @@ const AMP_EMBED_ALLOWED = {
   zergnet: true,
 };
 
-const data = parseFragment(location.hash);
-window.context = data._context;
-
+const iframeName = window.name;
+let data;
+try {
+  data = JSON.parse(window.name).attributes;
```

Comment:
I can fix this, but it would be a lot easier for me to do it in #6506 if that's alright with you. I need to make a helper function that constructs the whole name object anyways, and that's the PR where I'm going to put it